controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -270,15 +272,28 @@ func (r *StorageClusterReconciler) reconcileUninstallAnnotations(sc *ocsv1.Stora
 	return false, nil
 }
 
-// verifyNoStorageConsumerExist verifies there are no storageConsumers on the same namespace
-func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.StorageCluster) error {
+// verifyNoStorageConsumerExist verifies there are no storageConsumers on the same namespace conditionally
+func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.StorageCluster, skipLocalConsumer bool) error {
```

Comment:
There is no reason the have to skip the local consumer. 
The order of uninstallation should be StorageClient then StorageCluster so it is important to make sure we are NOT skipping it in the check

Reply:
After discussion we agreed to completely remove the check

Reply:
done.